lib/search/helpArticleSearch.ts

```
@@ -0,0 +1,41 @@
+  url: string;
+};
+
+export type SearchResult = HelpArticle;
+
+export function searchHelpArticles(articles: HelpArticle[], query: string, limit = 10): SearchResult[] {
+  const trimmedQuery = query.trim();
+  
+  if (!trimmedQuery) {
+    return articles
+      .sort((a, b) => a.title.localeCompare(b.title))
+      .slice(0, limit);
```

ESLint

[error] Replace `⏎······.sort((a,·b)·=>·a.title.localeCompare(b.title))⏎······` with `.sort((a,·b)·=>·a.title.localeCompare(b.title))`

(prettier/prettier)